cloudflare-ddns

```diff
@@ -153,6 +153,7 @@ def get_ipv6():
 def update_dns_record(auth, zone_id, record, ip_address):
     if record is None or ip_address is None:
         return
+    ip_address = ip_address.strip() # Fix bug with trailing whitespace, Fixes #39
```

Comment:
Hey @Snuggle,

thanks for your quick fix.
Can you add the strip method to line 129. Then the correct method would strip the ip address and the whole code could work with the correct/stripped ip, not just  `def update_dns_record()`